Assignment1/test-output/index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
```

Comment:
The `test-output` directory is not code/configuration, but output of your test output. Since it is generated from the code, you shouldn't add it to your code repository.